src/components/BurnerWalletCard/CardContainer.tsx

```diff
@@ -0,0 +1,63 @@
+import {
+  alignItems,
+  backgroundColor,
+  backgroundImage,
+  borderColor,
+  borderRadius,
+  borderWidth,
+  classnames,
+  display,
+  flexDirection,
+  gap,
+  gradientColorStops,
+  maxWidth,
+  padding,
+  position,
+  width,
+  wordBreak,
+  zIndex,
+} from 'classnames/tailwind'
+import ChildrenProp from 'models/ChildrenProp'
+import classNamesToString from 'helpers/classNamesToString'
+
+const cardContainer = classnames(
+  display('flex'),
+  flexDirection('flex-col'),
+  gap('gap-y-4'),
+  alignItems('items-center'),
+  position('relative'),
+  borderWidth('border'),
+  borderColor('border-transparent'),
+  borderRadius('rounded-2xl'),
+  padding('py-6', 'px-6', 'xs:py-10'),
+  wordBreak('break-words'),
+  zIndex('z-20')
+)
+const parentButtonWrapper = classnames(
```

Comment:
This file is called `CardContainer`, why this is `buttonWrapper`?

Reply:
Renamed